src/main/java/Command/AddCommand.java

```diff
@@ -0,0 +1,59 @@
+package command;
+
+import exceptions.InvalidCommandException;
+import exceptions.NoDateException;
+import exceptions.NoTaskException;
+import exceptions.WrongDateTimeFormatException;
+import parserstorageui.Storage;
+import parserstorageui.Ui;
+import task.TaskList;
+
+public class AddCommand extends Command {
+
+    /**
+     * Initializes AddCommand class
+     *
+     * @param command
```

Comment:
I think the recommended JavaDoc for methods with parameters is that you describe the parameter in a one-liner, so maybe "@param command command of user" would be better!